assets/queries/cloudFormation/amazonMQ_broker_without_encryption_options/query.rego

```diff
@@ -0,0 +1,18 @@
+package Cx
+
+CxPolicy [ result ] {
+  document := input.document
+  resource = document[i].Resources[name]
+  resource.Type == "AWS::AmazonMQ::Broker"
+  properties := resource.Properties
+  exists_eo := object.get(properties, "EncryptionOptions", "undefined") != "undefined"
+  not exists_eo
```

Comment:
simplify this changing line 8 and 9 to:
`object.get(properties, "EncryptionOptions", "undefined") == "undefined"`